engine/installation/linux/oracle.md

```diff
@@ -14,9 +14,13 @@ To get started with Docker on Oracle Linux, make sure you
 
 ### OS requirements
 
-To install Docker, you need the 64-bit version of Oracle Linux 6 or 7, running
-the Unbreakable Enterprise Kernel Release 4 (4.1.12) or higher. This kernel
-supports the Docker `btrfs` storage engine.
+To install Docker, you need the 64-bit version of Oracle Linux 6 or 7.
+
+To use `btrfs`, you need to install the Unbreakable Enterprise Kernel (UEK)
```

Comment:
This is not just required for `btrfs` support. Oracle will only support `docker-engine` when running the UEK4. Technically, `btrfs` is supported on both the UEK and RHCK kernel variants.

Though, I guess this is somewhat of a moot point, because the rest of the documentation outlines how to install a completely unsupported version of Docker anyway.